CHANGELOG.md

```diff
@@ -34,10 +24,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Misc
 
 ### Internal
-- Added an Internal section to the changelog. This section is intended to capture any notable non-public changes to the project.
+
 
 # Past Releases
 
+## [0.50.0]
```

Comment:
Can you date this? Seems we've been forgetting to do that for a while.

Reply:
Sure